flink-metric-prometheus/main.py

```diff
@@ -0,0 +1,110 @@
+#  Copyright 2022 The FeatHub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from datetime import timedelta
+
+from feathub.feathub_client import FeathubClient
+from feathub.feature_tables.sinks.black_hole_sink import BlackHoleSink
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.derived_feature_view import DerivedFeatureView
+
+from feathub.common import types
+from feathub.feature_tables.sources.file_system_source import FileSystemSource
+from feathub.table.schema import Schema
+from feathub.metric_stores.metric import Count, Ratio
+
+if __name__ == "__main__":
+    client = FeathubClient(
+        props={
+            "processor": {
+                "type": "flink",
+                "flink": {
+                    "master": "localhost:8081",
+                },
+            },
+            "online_store": {
+                "types": ["memory"],
+                "memory": {},
+            },
+            "registry": {
+                "type": "local",
+                "local": {
+                    "namespace": "default",
```

Comment:
how about changing this to `custom_namespace` so that the metric name is more self-explanatory?